mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb.inc
+
+if (`SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.ENGINES WHERE engine = 'rocksdb' AND support IN ('DEFAULT')`)
+{
+  --skip Test requires default engine RocksDB
+}
+
+# MYR-14
+#if (`select count(*) = 0 from information_schema.tables where engine='rocksdb' and table_name='slave_gtid_info'`)
```

Comment:
** Clarifying - facebook/webscale sql introduces a new system table slave_gtid_info that is part of their cash safe slave implementation. Since this table does not exist in Percona Server and we haven't made it to the point to decide what to do about MyRocks and crash safe slave, leaving this check enabled will cause all myrocks tests to be skipped. Commenting and tagging it with MYR-14 means that this code will be re-evaluated and corrected when we address MYR-14, crash safe slave and myocks.